deploy_capes.sh

```diff
@@ -1,4 +1,98 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+################################
+########## Globals #############
+################################
+
+set -o nounset  # Fail if we try to use an undeclared variable
+set -o errexit  # Fail if an error is encountered
+
+################################
+########### Options ############
+################################
+
+USE_FIREWALLD=true # Controls whether you use firewalld or iptables
+PORTAINER=true
+THEHIVE=true
+ROCKETCHAT=true
+MUMBLE=true
+ETHERPAD=true
+CYBERCHEF=true
+GITEA=true
+
+################################
+########## Versions ############
+################################
+
+MYSQL_VER=5.7
+HIVE_ELASTICSEARCH_VER=5.6.13
+ELASTICSEARCH_VER=7.0.1
+MONGODB_VER=4.0.9
+PORTAINER_VER=1.20.2
+NGINX_VER=mainline
+CYBERCHEF_VER=latest
+GITEA_VER=1.8.1
+ETHERPAD_VER=latest
+THEHIVE_VER=3.3.0
+ROCKETCHAT_VER=1.0.3
+MUMBLE_VER=v0.3
+
+################################
+########### Ports ##############
+################################
+
+NGINX_PORT=80
+ROCKETCHAT_INTERNAL_PORT=3000
+ROCKETCHAT_EXTERNAL_PORT=4000
+GITEA_PORT=3000
+ETHERPAD_EXTERNAL_PORT=5000
+ETHERPAD_INTERNAL_PORT=9001
+KIBANA_PORT=5601
+MUMBLE_PORT=64738
+CYBERCHEF_EXTERNAL_PORT=8000
+CYBERCHEF_INTERNAL_PORT=8080
+THEHIVE_PORT=9000
+PORTAINER_INTERNAL_PORT=9000
+PORTAINER_EXTERNAL_PORT=2000
+ELASTICSEARCH_PORT=9200
+ELASTICSEARCH_MANAGEMENT_PORT=9300
+
+################################
+###### Helper Functions ########
+################################
+
+#######################################
+# Delete a container if it already exists
```

Comment:
I don't think I want to do this. I like the idea of understanding the environment and making good decisions based on that, but I don't know that I want to delete containers.

Reply:
PR Updated